client/src/components/Chat/FileItem/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { FileItemWrapper } from './style';
+
+function FileItem({ src, alt, itemType }: { src: string; alt: string; itemType: string }) {
```

Comment:
```suggestion

interface Props {
  src: string;
  alt: string;
  itemType: string;
}

function FileItem({ src, alt, itemType }: Props) {
```

prop 인터페이스 따로 빼도 좋을 것 같어유!